wiki/People/The_Team/Nomination_Assessment_Team/en.md

```diff
@@ -9,35 +9,63 @@ tags:
+- **Modding:** Modding at least 3 beatmaps per month. This helps evaluators to keep up to date with the mapping/modding community when evaluating current and aspiring Beatmap Nominators.
 
-The NAT are responsible for a variety of mapping-related tasks that can be separated into four subcategories:
+Evaluators are in high demand due to the sheer volume of [BN applications](/wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator). They, unlike advisors, therefore occasionally look for new members to join them and help ease the workload. This is why BNs that wish to join the NAT are primarily judged based on their ability to evaluate, and most NAT members join and stay as evaluators.
 
-- **Evaluation:** Evaluating the proficiency and activity of both current Beatmap Nominators and applicants. Most NAT members are responsible for contributing to this subcategory. For full details on how evaluations work, please see the [evaluations](/wiki/People/The_Team/Nomination_Assessment_Team/Evaluations) page.
-- **Structural:** Documenting topics surrounding the ranking process, amending rules and guidelines to the [ranking criteria](/wiki/Ranking_Criteria), developing and maintaining related tools and websites.
-- **Moderation:** Handling user reports and assessing inappropriate behaviour of Beatmap Nominators. This subcategory is a joint effort between the NAT and the GMT.
-- **Communication:** Promoting transparency between the NAT and the mapping/modding community, including the creation and hosting of meetings.
+### Advisors
 
-A member of the NAT can engage with any tasks within any subcategory that they choose, despite what their primary responsibilities may be listed as. For example, if a member of the NAT prioritises the Structural subcategory, they are not responsible for all tasks within the subcategory and they can still help with tasks from any other subcategory.
+As a group, **Advisors** are required to maintain:
 
-In addition to moderation abilities, NAT also possess the ability to [nominate](/wiki/Beatmap_ranking_procedure#nominations), [qualify](/wiki/Beatmap_ranking_procedure#qualification), and [reset nominations](/wiki/Beatmap_ranking_procedure#nomination-resets) on beatmaps. In this area they are considered largely the same as [full BNs](/wiki/People/The_Team/Beatmap_Nominators#full-beatmap-nominators), and all BN rules and expectations besides activity apply to NAT members. NAT members are not expected to nominate maps regularly the way BNs are due to having additional responsibilities in the osu! mapping and modding community.
+- **Communication:** Promoting transparency and maintaining good relations with not only the rest of the mapping/modding community, but also within the NAT itself. This includes, but not limited to, making announcements, participating in discussions about proposals, asking/answering questions through surveys, and updating documentation such as the [Ranking Criteria](/wiki/Ranking_Criteria).
```

Comment:
what "proposals"? RC proposals?

```suggestion
- **Communication:** Promoting transparency and maintaining good relations with not only the rest of the mapping/modding community, but also within the NAT itself. This includes, but not limited to, making announcements, participating in discussions about proposals, asking/answering questions through surveys, and updating the [ranking criteria](/wiki/Ranking_Criteria) as well as other documentation.
```

Reply:
basically anything in [rc subform](https://osu.ppy.sh/community/forums/87) (its just that they're not all technically about the rc specifically) + internal proposals